chart/templates/triggerer/triggerer-deployment.yaml

```diff
@@ -160,6 +163,7 @@ spec:
         - name: triggerer
           image: {{ template "airflow_image" . }}
           imagePullPolicy: {{ .Values.images.airflow.pullPolicy }}
+          securityContext: {{ $containerSecurityContext | nindent 12 }}
```

Comment:
`triggerer-log-groomer` is missing `securityContexts` implementation

Reply:
Thanks for spotting: I tried to add https://github.com/apache/airflow/pull/24588/files#diff-747f774dc1498138f6e6b9b4ab1fef11473ef9e05ee8ba85e2a3755ebd114ac3R225 to fix it